test/e2e/pageobjects/app-frontend.ts

```diff
@@ -13,6 +13,9 @@ export class AppFrontend {
 
     /** @see https://dev.altinn.studio/repos/ttd/signing-test */
     signingTest: 'signing-test',
+
+    /** @see https://dev.altinn.studio/repos/ttd/expression-validation-test */
+    expressionValidationTest: 'expression-validation-test',
```

Comment:
You should add this in the list of apps we test in the `README.md` as well. 🙌

Reply:
👍